skops/saving_utils/model_card.py

```diff
@@ -0,0 +1,55 @@
+from modelcards import CardData, ModelCard
+from sklearn import set_config
+from sklearn.utils import estimator_html_repr
+
+
+def _plot_estimator(model):
+    set_config(display="diagram")
+    return estimator_html_repr(model)
+
+
+def _extract_estimator_config(model):
+    """Extracts estimator configuration and renders them into a vertical table.
+
+    Args:
+        model (estimator): scikit-learn pipeline or model.
+
+    Returns:
+        str: Markdown table of hyperparameters.
```

Comment:
This repo uses `rst` format with numpydoc style. This section would be:

```
    Parameters
    ----------
    model: estimator
        scikit-learn compatible estimator.

    Returns
    -------
    str:
        Markdown table of hyperparameters.
```

Reply:
I did pre-commit install which I thought would automatically format things but apparently it's not the case 😂